### PR TITLE
local build wipe off source path

### PR DESCRIPTION
**Pull request by carlos-lee**
### Describe what this PR does / why we need it

local build fit this Kubefile：
FROM registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.19.9
COPY /Users/zcy/Documents/sealer-src/recommended.yaml .
CMD kubectl apply -f recommended.yaml

### Describe how you did it
 wipe off copy file source path

### Describe how to verify it
local build success

### CODE REVIEW

File: build/local_builder.go [hunk 1]
```diff
@@ -256,7 +257,7 @@ func (l *LocalBuilder) execLayer(layer *v1.Layer, tempTarget string) error {
 			dist = filepath.Join(tempTarget, strings.Fields(layer.Value)[1])
 		} else {
 			// src is file
-			dist = filepath.Join(tempTarget, strings.Fields(layer.Value)[1], strings.Fields(layer.Value)[0])
+			dist = filepath.Join(tempTarget, strings.Fields(layer.Value)[1], path.Base(strings.Fields(layer.Value)[0]))
```
Comment (fanux) [inline build/local_builder.go:260]: I think we can't using source path here, We should match docker's copy behavior.

**Comment by fanux** [comment]
@kakaZhou719 the sourpath should be `context/layer[0]`

**Comment by carlos-lee** [comment]
modify README.md,better  UX

**Comment by cuisongliu** [comment]
please use English  for this pr.

File: README.md [hunk 2]
```diff
@@ -38,8 +38,8 @@ Kubefile:
...
...XBlcjp2MS4wLjYKICAgICAgICAgIHBvcnRzOgogICAgICAgICAgICAtIGNvbnRhaW5lclBvcnQ6IDgwMDAKICAgICAgICAgICAgICBwcm90b2NvbDogVENQCiAgICAgICAgICBsaXZlbmVzc1Byb2JlOgogICAgICAgICAgICBodHRwR2V0OgogICAgICAgICAgICAgIHNjaGVtZTogSFRUUAogICAgICAgICAgICAgIHBhdGg6IC8KICAgICAgICAgICAgICBwb3J0OiA4MDAwCiAgICAgICAgICAgIGluaXRpYWxEZWxheVNlY29uZHM6IDMwCiAgICAgICAgICAgIHRpbWVvdXRTZWNvbmRzOiAzMAogICAgICAgICAgdm9sdW1lTW91bnRzOgogICAgICAgICAgLSBtb3VudFBhdGg6IC90bXAKICAgICAgICAgICAgbmFtZTogdG1wLXZvbHVtZQogICAgICAgICAgc2VjdXJpdHlDb250ZXh0OgogICAgICAgICAgICBhbGxvd1ByaXZpbGVnZUVzY2FsYXRpb246IGZhbHNlCiAgICAgICAgICAgIHJlYWRPbmx5Um9vdEZpbGVzeXN0ZW06IHRydWUKICAgICAgICAgICAgcnVuQXNVc2VyOiAxMDAxCiAgICAgICAgICAgIHJ1bkFzR3JvdXA6IDIwMDEKICAgICAgc2VydmljZUFjY291bnROYW1lOiBrdWJlcm5ldGVzLWRhc2hib2FyZAogICAgICBub2RlU2VsZWN0b3I6CiAgICAgICAgImt1YmVybmV0ZXMuaW8vb3MiOiBsaW51eAogICAgICAjIENvbW1lbnQgdGhlIGZvbGxvd2luZyB0b2xlcmF0aW9ucyBpZiBEYXNoYm9hcmQgbXVzdCBub3QgYmUgZGVwbG95ZWQgb24gbWFzdGVyCiAgICAgIHRvbGVyYXRpb25zOgogICAgICAgIC0ga2V5OiBub2RlLXJvbGUua3ViZXJuZXRlcy5pby9tYXN0ZXIKICAgICAgICAgIGVmZmVjdDogTm9TY2hlZHVsZQogICAgICB2b2x1bWVzOgogICAgICAgIC0gbmFtZTogdG1wLXZvbHVtZQogICAgICAgICAgZW1wdHlEaXI6IHt9Cg==" |base64  -d  > recommended.yaml
```
Comment (fanux) [inline README.md:42]: OH, put those text in README is Terrible. I think you can add a dashboard application: [helm example](https://github.com/alibaba/sealer/tree/main/applications/helm/v3.6.0)

Comment (carlos-lee) [reply README.md:42]: OK, I try sealer follow readme found this 
 url "https://raw.githubusercontent.com/kubernetes/dashboard/v2.2.0/aio/deploy/recommended.yaml" sometimes timeout, so I think change offline type for better experience

**Pull request closed without merging**